homeassistant/components/media_player/firetv.py

```diff
@@ -166,6 +171,8 @@ def supported_features(self):
     @property
     def state(self):
         """Return the state of the player."""
+        if not self._available:
```

Comment:
We don't need to check this if we use it for available property. Then the base entity will handle that.

Reply:
Does that apply to the `app_id`, `source`, and `source_list` properties as well?

Reply:
It applies to all properties that are part of state attributes. It doesn't apply to unit of measurement, name, entity_picture, hidden, icon, assumed_state, supported_features and device_class.

Reply:
Fixed in https://github.com/home-assistant/home-assistant/pull/21221/commits/e9f139128c57a7062e965a13b3ac299e8acd3c10